package.json

```diff
@@ -23,6 +23,7 @@
   "devDependencies": {
     "babel-brunch": "~6.0.0",
     "babel-preset-es2015": "^6.6.0",
+    "babel-preset-es2016": "^6.16.0",
```

Comment:
I'm not positive, but I want to say that you do. I don't think es2016 includes all that es2015 did.